### PR TITLE
Calculator test

### PR DESCRIPTION
**Pull request by VyacheslavBukach**

### CODE REVIEW

File: test/calculator_test.dart [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:my_books/calculator.dart';
+
+void main() {
```
Comment (MaximSemashko) [inline test/calculator_test.dart]: discussed offline; please correct all raised issues

File: lib/triangle_helper.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    required this.secondSide,
+    required this.thirdSide,
+    required this.height,
+  });
+
+  double findSquare() {
+    var result1 = calculator.divide(number1: 1, number2: 2);
+    var result2 = calculator.multiply(number1: result1, number2: thirdSide);
+    var result3 = calculator.multiply(number1: result2, number2: height);
+    return result3;
```
Comment (MaximSemashko) [inline lib/triangle_helper.dart:19]: add empty line before return

File: lib/triangle_helper.dart [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  double findPerimeter() {
+    var result1 = calculator.sum(number1: oneSide, number2: secondSide);
+    var result2 = calculator.sum(number1: result1, number2: thirdSide);
+    return result2;
+  }
```
Comment (MaximSemashko) [inline lib/triangle_helper.dart:27]: add empty line before return

File: test/calculator_test.dart [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  late Calculator calculator;
+
+  setUp(() {
+    calculator = Calculator();
+  });
+
+  test('Sum_Test', () {
```
Comment (MaximSemashko) [inline test/calculator_test.dart]: you can write just 'sum test', 'division test' etc. Please correct

File: test/calculator_test.dart (hunk 1 above)
Comment (VyacheslavBukach) [reply test/calculator_test.dart]: done

File: lib/triangle_helper.dart (hunk 2 above)
Comment (VyacheslavBukach) [reply lib/triangle_helper.dart:19]: done

File: lib/triangle_helper.dart (hunk 3 above)
Comment (VyacheslavBukach) [reply lib/triangle_helper.dart:27]: done

File: test/calculator_test.dart (hunk 4 above)
Comment (VyacheslavBukach) [reply test/calculator_test.dart]: done

**Pull request merged**